docs/hint.md

```diff
@@ -0,0 +1,121 @@
+# Hint widget
+
+Display hint for field form.
+
+## Usage
+
+```php
+<?php
+
+declare(strict_types=1);
+
+namespace App\Form;
+
+use Yiisoft\Form\FormModel;
+
+final class TestForm extends FormModel
+{
+    private string $name = '';
+
+    public function getAttributeHints(): array
+    {
+        return [
+            'name' => 'Write your first name.',
+        ];
+    }    
+}
+```
+
+Widget view:
+
+```php
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Form\FormModelInterface;
+use Yiisoft\Form\Widget\Field;
+use Yiisoft\Form\Widget\Form;
+use Yiisoft\Form\Widget\Hint;
+use Yiisoft\Form\Widget\Text;
+
+/**
+ * @var FormModelInterface $data
+ * @var object $csrf
+ */
+?>
+
+<?= Form::widget()->action('widgets')->csrf($csrf)->begin() ?>
+    <?= Text::widget()->config($formModel, 'name') ?>
+    <?= Hint::widget()->config($formModel, 'name') ?>
```

Comment:
Just a suggestion, but it looks interesting:

```suggestion
    <?= Text::widget()->for($formModel, 'name') ?>
    <?= Hint::widget()->for($formModel, 'name') ?>
```

Reads as "hint widget for $model's "name".

Reply:
But that conflicts with https://github.com/yiisoft/form/pull/146/files#diff-b2dbc04239858c959223583f766058bc1685ade0b8ff4c9278cd3d30866bd5aaR119 :( Need a better name then...